ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml

```diff
@@ -135,3 +135,42 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    branches:
+    - ^main$
+    cluster: build02
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: branch-ci-open-cluster-management-clusterlifecycle-state-metrics-main-sonarcloud
+    spec:
+      containers:
+      - args:
+        - -f
+        - /opt/build-harness/Makefile.prow
+        - sonar/go/prow
+        command:
+        - make
+        env:
+        - name: ARTIFACT_DIR
+          value: $(ARTIFACTS)
+        - name: GOPROXY
+          value: proxy.golang.org,direct
+        - name: SELF
+          value: make -f /opt/build-harness/Makefile.prow
+        image: registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux
```

Comment:
I think it should be 1.16 as the dockerfile.prow is using go1.16

Reply:
updated